### ISSUE TITLE
image has dip switches in wrong position

### ISSUE THREAD

**Issue by bbrendon** [adonno/tagreader#233]
it seems this image has the dip switches in the wrong position?
https://github.com/adonno/tagreader/blob/master/docs/cases.jpg?raw=true

**Comment by adonno** [comment]
Indeed, This is an older image when we where using SPI instead of I2C
